config/locales/views/settings/en.yml

```diff
@@ -31,7 +31,7 @@ en:
             Delete your profile, along with your authentication associations.
 
             This does not include applications permissions. You will have to remove them yourself:
-          desc1b: Delete any and all content you have, such as articles, comments, your reading list or chat messages.
```

Comment:
Thanks for the cleanup here after the connect chat removal!

Reply:
These are reinstatement of #14734 which has overtaken the corresponding i18n PR.